core/mm/core_mmu.c

```diff
@@ -255,6 +259,47 @@ static struct tee_mmap_region *find_map_by_pa(unsigned long pa)
 	return NULL;
 }
 
+#if defined(CFG_SECURE_DATA_PATH)
+static bool dtb_get_sdp_region(void)
+{
+	void *fdt = NULL;
+	int node = -1;
+	paddr_t tmp_addr = 0;
+	size_t tmp_size = 0;
+
+	if (!IS_ENABLED(CFG_EMBED_DTB))
+		return false;
+
+	fdt = get_embedded_dt();
+	if (!fdt)
+		panic("No DTB found");
+
+	node = fdt_node_offset_by_compatible(fdt, node, tz_sdp_match);
```

Comment:
nitpicking: prefer use an explicit `-1` 2nd argument rather than `node` inited to -1.

Or even better, use a loop to support several compatible nodes if found, and create an array of `struct core_mmu_phys_mem`. But maybe in a later change if you want this initial support to be merged before coming 3.18.0 tag.